### PR TITLE
Add a Code of Conduct

### LINKED ISSUE #1: Add code of conduct to project
Write and add the code of conduct
https://help.github.com/en/github/building-a-strong-community/adding-a-code-of-conduct-to-your-project

### LINKED ISSUE #2: Research code of conduct

### PR DESCRIPTION
**Pull request by danisquierdo**
Closes #1 
Closes #2 

Adding code of conduct v1

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+
+## 7. Reporting Guidelines
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. Reporting Form: https://docs.google.com/forms/d/e/1FAIpQLSfvv6TygQZ9afS7Vy1UdUSpBF99xMMuPmKvJITm5VR-Cfa9Og/viewform?usp=sf_link.
+
+
+
+Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe. In the context of in-person events, organizers will also provide escorts as desired by the person experiencing distress.
```
Comment (arbitraryconstants) [inline CODE_OF_CONDUCT.md]: I would axe this line.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+
+## 7. Reporting Guidelines
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. Reporting Form: https://docs.google.com/forms/d/e/1FAIpQLSfvv6TygQZ9afS7Vy1UdUSpBF99xMMuPmKvJITm5VR-Cfa9Og/viewform?usp=sf_link.
```
Comment (arbitraryconstants) [inline CODE_OF_CONDUCT.md]: > as soon as possible... either in person or through this reporting form: <link>

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+
+
+Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe. In the context of in-person events, organizers will also provide escorts as desired by the person experiencing distress.
+
+## 8. Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify Philly Coffee and Code with a concise description of your grievance. Your grievance will be handled in accordance with our existing governing policies.
```
Comment (arbitraryconstants) [inline CODE_OF_CONDUCT.md]: I would axe: `Your grievance will be handled in accordance with our existing governing policies. `

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+## 1. Purpose
+
+A primary goal of Philly Coffee and Code is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
+
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+
+We invite all those who participate in Philly Coffee and Code to help us create safe and positive experiences for everyone.
+
+## 2. Open [Source/Culture/Tech] Citizenship
```
Comment (arbitraryconstants) [inline CODE_OF_CONDUCT.md]: I'm not sure if this section is necessary.

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+# Citizen Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Philly Coffee and Code is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```
Comment (arbitraryconstants) [inline CODE_OF_CONDUCT.md]: How about something like...?


> A primary goal of Philly Coffee and Code is to bring a diverse group of people together to work on projects and to socialize. We aim to achieve this by hosting monthly Coffee and Code events. 

> These Coffee and Codes are intended to be pretty casual. You can work on your own thing, create stuff with other participants, or just hangout and socialize --  up to you!

> We are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).

**Comment by arbitraryconstants** [comment]
Hey @danisquierdo, I've reopened this issue and added some feedback regarding content and wording. Let me know what you think! ✨

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (danisquierdo) [reply CODE_OF_CONDUCT.md]: I think the line "These Coffee and Codes are intended to be pretty casual. You can work on your own thing, create stuff with other participants, or just hangout and socialize -- up to you!" is great but more to be included in our description of the meetup? Not necessarily for the CoC, but we can have it in both

Comment (arbitraryconstants) [reply CODE_OF_CONDUCT.md]: Good point. I'll remove that line.

**Comment by arbitraryconstants** [comment]
This document will likely need updating but this seems like a good start :)

**Pull request merged**